com.google.Chrome.appdata.xml

```diff
@@ -39,18 +39,18 @@
 <summary>Google's popular web browser</summary>
 
 
-<description xml:lang="ar"><p>Google Chrome هو أحد أشهر مستعرضات الإنترنت في العالم. إذا واجهتك أية مشاكل مع المستعرضات الأخرى أثناء المشاهدة المباشرة للفيديوهات عبر الإنترنت، جرِّبْ Google Chrome، حيث يوجد به دعم مدمج لمعظم المواقع الإلكترونية التي تقدم خدمة البث المباشر للفيديوهات. ** يتطلب اتصال بالإنترنت</p></description>
```

Comment:
I guess you used a translator for AR, since the "require internet" is on the left of the asterisks.

Reply:
Actually I just guessed.

Reply:
Or rather, the text editor doesn't reverse the text direction so it was to the right of the asterisks.